README.md

```diff
@@ -163,15 +163,38 @@ Configuration information is specified in two forms:
-| `keystore_type`           | `NA`                | Set the key store type                                                    |
-| `keystore_provider`       | `NA`                | Set the provider name of the key store                                    |
-| `keystore_path`           | `NA`                | Set the path to the key store file                                        |
-| `keystore_password`       | `NA`                | Set the password for the key store                                        |
-| `key_alias`               | `NA`                | Optional param that points to a specific key within the key store         |
-| `key_alias_password`      | `NA`                | Optional param that points to a password of `key_alias` if it protected   |
 | `api_key_sources`         | `PARAM,HEADER,PATH` | Defines the sources (order of precendence) of the API key.                |
 
+### System Properties for TLS configuration for Http server
+To configure Transport Layer Security (TLS) for HTTP server in edge module, the following configuration parameters should be used.
+Parameters marked as Required are required only in case when ssl_enabled is set to true.
+
+| Property                          | Default           | Description                                                                                 |
+|-----------------------------------|-------------------|---------------------------------------------------------------------------------------------|
+| `http-server.ssl_enabled`         | `false`           | Set whether SSL/TLS is enabled for Vertx Http Server                                        |
+| `http-server.keystore_type`       | `NA`              | (Required). Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS` |
+| `http-server.keystore_provider`   | `NA`              | Set the provider name of the key store                                                      |
+| `http-server.keystore_path`       | `NA`              | (Required). Set the location of the keystore file in the local file system                  |
+| `http-server.keystore_password`   | `NA`              | (Required). Set the password for the keystore                                               |
+| `http-server.key_alias`           | `NA`              | Set the alias of the key within the keystore.                                               |
+| `http-server.key_alias_password`  | `NA`              | Optional param that points to a password of `key_alias` if it protected                     |
+
+### System Properties for TLS configuration for Web Client
+To configure Transport Layer Security (TLS) for Web clients in the edge module, you can use the following configuration parameters.
+Truststore parameters for configuring Web clients are optional even when ssl_enabled = true.
+If truststore parameters need to be populated - truststore_type, truststore_path, truststore_password - are required.
```

Comment:
This sentence could be: `If truststore parameters need to be populated, truststore_type, truststore_path and truststore_password are required.`

Reply:
improved, thanks